src/styles/blocks/card-block.scss

```diff
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  font-style: $main-font-style;
+  color: $main-text-color;
+
+  &__img {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img-imac {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__texts-inform {
+    margin-top: 40px;
+  }
+
+  &__title {
+    margin: 0;
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $second-text-color;
+  }
+
+  &__stars-reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
```

Comment:
The class name `&__stars-reviews` should follow the BEM naming convention. Consider renaming it to `.container__stars-reviews` or similar.